docs/ml/creating-df-kib.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[creating-df-kib]]
+== Creating {dataframe-transforms}
+
+beta[]
+
+If you want to use the {dataframe-transforms} feature, you must have the required 
+user roles and permissions:
+
+* `data_frame_transforms_user`: to get transforms and transforms stats,
+* `data_frame_transforms_admin`: to create/start/stop/delete transforms,
+* read permissions on the source indices,
+* index permissions on the target index (if the target index does not exist, you 
+need permission to create the index as well).
+
+[role="screenshot"]
+image::ml/images/ml-definepivot.jpg["Defining a {dataframe} pivot"]
+
+You can create {dataframe-transforms} in the {kib} Machine Learning application. 
+Select an index or a search pattern you want to transform and a field that the data 
+will be grouped-by, then define the desired aggregation. The {dataframe} pivot 
```

Comment:
You can select one or more fields to group by. Maybe you want to say `and one or more fields that the data will be grouped by`, or similar? Also I'd use `grouped by` rather than hyphenating as `grouped-by`. Same applies to aggregations, so maybe `desired aggregations` rather than just `aggregation`?

Reply:
Maybe something along the lines of this?: To pivot your data, you must group the data by at least one field and apply at least one aggregation.